widget_driver_generator/lib/src/model_visitor.dart

```diff
@@ -1,22 +1,40 @@
 import 'package:analyzer/dart/element/element.dart';
 import 'package:analyzer/dart/element/visitor.dart';
 import 'package:widget_driver_annotation/widget_driver_annotation.dart';
+import 'package:widget_driver_generator/src/utils/providable_field.dart';
 
 import 'utils/code_writer.dart';
 import 'utils/source_code_generator.dart';
 import 'utils/element_utils.dart';
 
 typedef CodeGeneratorMethod = String Function(String codeDefinition, String returnValue);
 
-/// Inspect classes and get className and fields out of them
+/// Inspect classes and get className, fields and all constructor parameters annotated with the @driverProvidableFields.
 class ModelVisitor extends SimpleElementVisitor<void> {
   String className = "";
   final fields = <String, dynamic>{};
+  final providableFields = <ProvidableField>[];
+  final ElementUtils _elementUtils;
+
+  ModelVisitor({ElementUtils? elementUtils})
```

Comment:
```suggestion
  const ModelVisitor({ElementUtils? elementUtils})
```

Reply:
If I'm not mistaken, `fields` and `providableFields` are non-const values (lists), so Model Visitor can't be `const`.